UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -747,9 +812,14 @@
         var allProperties = selectedMassTransitStopModel.getProperties();
         var properties;
 
-        if (isTerminalBusStop) {
+        /* don't change == to ===
+        * sometimes the value is int other times is string*/
+        if (isTerminalBusStop == 6 ) {
```

Comment:
Change the variable name to something more descriptive, it doesn't make sense to compare isSomething with a number

Reply:
Changed variable name to busStopTypeSelected